cmd/inspektor-gadget/cmd/bcck8s.go

```diff
@@ -81,6 +85,41 @@ func init() {
 	capabilitiesCmd.PersistentFlags().BoolVarP(&verboseFlag, "verbose", "", false, "Include non-audit")
 }
 
+type postProcess struct {
+	nodeName string
+	nodeShort string
+	orig io.Writer
+	firstLine bool
+	firstLinePrinted *uint64
+	failure chan string
+}
+
+func (post postProcess) Write(p []byte) (n int, err error) {
+	prefix := "[" + post.nodeShort + "] "
+	asStr := ""
+        if len(p) != 0 {
+		asStr = string(p)
```

Comment:
We don't seem to need to test len(p) and we can just write `asStr := string(p)`:
https://play.golang.org/p/3xsKw7KnXgE

Reply:
Thanks, yes, changed.